src/screens/SettingsScreen.tsx

```diff
@@ -342,30 +355,30 @@ export class SettingsScreen extends React.Component<Props, State> {
                 </View>
               </View>
             </RNPickerSelect>
+            <SwitchWithText
+              onValueChange={this._handleToggleOfflineMode}
+              subText={translate('Offline mode can save battery life and improve performance')}
+              text={translate('Offline Mode')}
+              value={!!offlineModeEnabled}
+            />
             <Divider style={styles.divider} />
             <Button
               onPress={this._handleClearHistory}
               wrapperStyles={styles.button}
               text={translate('Clear History')}
             />
-
-            <Divider style={styles.divider} />
             <Button
               onPress={this._handleToggleDeleteDownloadedEpisodesDialog}
               wrapperStyles={styles.button}
               text={translate('Delete Downloaded Episodes')}
             />
-
             {isLoggedIn && (
-              <View>
-                <Divider style={styles.divider} />
-                <Button
-                  isWarning={true}
-                  onPress={this._handleToggleDeleteAccountDialog}
-                  text={translate('Delete Account')}
-                  wrapperStyles={styles.button}
-                />
-              </View>
+              <Button
```

Comment:
Is this button needed in Podable?

Reply:
No, good catch...how about we add a DISABLE_LOGIN env var and use it disable this and all logged in user features?

Reply:
Yes I think I agree. Better to make it configurable than not :)